### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by Yehorf21**
- [DEMO LINK](https://Yehorf21.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Yehorf21.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,388 @@
...
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__links is-active" href="index.html">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```
Comment (volodymyr-soltys97) [inline src/index.html:39]: Add the blank line between the links for better readability
```suggestion
          </li>
          
          <li class="nav__item">
```

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,388 @@
...
+      </nav>
+    </header>
+
+    <main class="catalog__main">
+      <div class="catalog__wrapper">
+        <div class="card" data-qa="card">
+          <img
+          class="card__img"
+          src="/src/images/imac.jpeg"
+          alt="imac"></img>
```
Comment (volodymyr-soltys97) [inline src/index.html:88]: Fix it everywhere
```suggestion
          <img
            class="card__img"
            src="/src/images/imac.jpeg"
            alt="imac"
          />
```

File: src/index.html [hunk 3]
```diff
@@ -5,8 +5,388 @@
...
+            </p>
+            <p class="card__price-value">
+              $2,199
+            </p>
+          </div>
+          <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+          >
```
Comment (volodymyr-soltys97) [inline src/index.html:119]:
```suggestion
          <a
            href="#"
            class="card__buy"
            data-qa="card-hover"
          >
```

File: src/index.html [hunk 4]
```diff
@@ -5,8 +5,388 @@
...
+          buy
+          </a>
+        </div>
+        <div class="card" data-qa="card">
```
Comment (volodymyr-soltys97) [inline src/index.html:123]: Add the blank line between the cards
```suggestion
        </div>
        
        <div class="card" data-qa="card">
```

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price-key {
+    color: $secondary-color;
+    font-size: 12px;
+    font-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/card.scss:71]: Default value, so you can just remove it everywhere
```suggestion
```

File: src/styles/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    line-height: 18px; /* 150% */
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/card.scss:72]: Remove comments everywhere
```suggestion
    line-height: 18px;
```

File: src/styles/blocks/header.scss [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+  }
+
+}
+
+
+
+
+
+
+a.is-active {
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/header.scss:54]:
```suggestion
.is-active {
```

Comment (volodymyr-soltys97) [inline src/styles/blocks/header.scss:54]:
```suggestion
}

a.is-active {
```

File: src/styles/utils/variables.scss [hunk 8]
```diff
@@ -0,0 +1,7 @@
+$main-accent: #060B35;
+$secondary-color: #616070;
+$blue-accent: #00ACDC;
+$border-color: #F3F3F3;
+$background-color: #FFF;
+$link-text-color: #FFF;
```
Comment (volodymyr-soltys97) [inline src/styles/utils/variables.scss:6]: Why did you create two variables with the same values?
Also white color is default color
```suggestion
$background-color: #FFF;
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
The card after the hover should decrease as smoothly as it increases, it needs to be fixed
image

File: src/styles/utils/variables.scss (hunk 8 above)
Comment (Yehorf21) [reply src/styles/utils/variables.scss:6]: When I was copying variables from another task, I forgot to remove the duplicates. Will be fixed

File: src/index.html [hunk 9]
```diff
@@ -5,8 +5,414 @@
...
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+
+          <li class="nav__item">
```
Comment (IvanFesenko) [inline src/index.html:36]: Empty line adds for siblings except first and last
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
```

File: src/index.html [hunk 10]
```diff
@@ -5,8 +5,414 @@
...
+            <p class="card__price-value">
+              $2,199
+            </p>
+          </div>
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+          buy
```
Comment (IvanFesenko) [inline src/index.html:132]:
```suggestion
            buy
```

File: src/styles/blocks/card.scss [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+  grid-gap: 46px 48px;
+  padding-bottom: 72px;
+  padding-block: 50px;
+  padding-inline: 40px;
+
+  @include columns;
+}
+
+.card {
+  box-sizing: border-box;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:13]: move ` box-sizing: border-box;` into `*` selector
 ```suggestion

```

File: src/styles/blocks/card.scss [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    border-radius: 5px;
+    background-color: $blue-accent;
+    text-decoration: none;
+    color: $link-text-color;
+
+    &:hover {
+      box-sizing: border-box;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:96]:
```suggestion

```

File: src/styles/blocks/header.scss [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+    padding: 0;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__links {
+    display: inline-block;
+    height: 60px;
```
Comment (IvanFesenko) [inline src/styles/blocks/header.scss:32]: do not hard code height - use padding instead

File: src/styles/blocks/header.scss [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+    line-height: 60px;
+    color: $main-accent;
+    text-decoration: none;
+    font-family: Roboto, sans-serif;
```
Comment (IvanFesenko) [inline src/styles/blocks/header.scss:36]:
```suggestion
    font-family: inherit;
```

File: src/styles/blocks/stars.scss [hunk 15]
```diff
@@ -0,0 +1,18 @@
...
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+  }
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(/src/images/star-active.svg);
+}
```
Comment (IvanFesenko) [inline src/styles/blocks/stars.scss:18]: should works with modifiers 1-5

File: src/styles/main.scss [hunk 16]
```diff
@@ -1,3 +1,10 @@
...
+@import '/src/styles/utils/variables';
+@import '/src/styles/utils/mixins';
+@import '/src/styles/blocks/page' ;
+@import '/src/styles/blocks/header';
+@import '/src/styles/blocks/card';
+@import '/src/styles/blocks/stars';
+
+
+
+
```
Comment (IvanFesenko) [inline src/styles/main.scss:10]:
```suggestion
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Some thing wrong with deploy - please re check demo link before re-request review
2024-01-29_15h09_42

**Pull request closed without merging**